.prettierrc

```diff
@@ -1,5 +0,0 @@
-{
-  "trailingComma": "all",
-  "singleQuote": true,
```

Comment:
These overrides were a clue that our shared config wasn't being used, since the shared config has the same values.